metrics/cli/docs/features/update/06_update_feature.md

```diff
@@ -0,0 +1,139 @@
+# Update feature
+> Feature description / User story.
+
+As a user, I want to have the opportunity to update an existing project, so I don't have to create a new one every time.
+
+## Contents
+
+- [**Analysis**](#analysis)
+    - [Feasibility study](#feasibility-study)
+    - [Requirements](#requirements)
+    - [Landscape](#landscape)
+    - [Prototyping](#prototyping)
+    - [System modeling](#system-modeling)
+
+# Analysis
+> Describe a general analysis approach.
+
+During the analysis stage, we are going to investigate the implementation approaches for updating existing projects with
+the Metrics CLI.
+
+### Feasibility study
+> A preliminary study of the feasibility of implementing this feature.
+
+The `Update feature` expands the end-user capabilities and improves the UX of the Metrics CLI, so this feature
+makes sense for the project.
+
+Moving on to the feasibility of implementation, the `Update feature` requires the following actions:
+
+- parsing the YAML configuration file;
+- redeploying/updating the existing Metrics Web application to the same Firebase project.
+
+The following proofs cover the above requirements, so we can implement the `Update feature`:
+- to parse the YAML configuration file, we can use a [package](https://github.com/Flank/flank-dashboard/tree/master/yaml_map) that applies in the ci_integrations module for the same purposes;
+- the Metrics CLI has everything we need to perform the redeploy action to the same Firebase project since the CLI can perform a deploy action.
+
+### Requirements
+> Define requirements and make sure that they are complete.
+
+Let's start with visiting and analyzing resources to collect the necessary data during the analysis stage.
+Thanks to these data, we can create precise and determined requirements and edge cases for the `Update feature`.
+
+The `Update feature` has the following requirements:
+- should read the configuration from the YAML configuration file;
+- should consume the configuration file path via the command line arguments;
+- should validate the YAML configuration file;
+- should stop the redeploy process of the Metrics CLI if something went wrong during the process;
+- should redeploy/update the existing Metrics Web application to the same Firebase project;
+- configuring the Sentry should be optional.
+
+### Landscape
+> Look for existing solutions in the area.
+
+There are no ready packages or solutions that satisfy our requirements since the `Update feature` is a custom feature 
+necessary exclusively for Metrics specific purposes in the Metrics CLI tool. Considering all the previous, custom implementation is the best choice.
```

Comment:
In the Landscape section, we search not only packages that may or may not satisfy our requirements. We also refer to the existing solutions that are similar to the feature under analysis. In our case, we can refer to the Firebase CLI that updates the existing project if the deployment is being performed for the already deployed components. For example, the `firebase deploy --only firestore` updates Cloud Firestore security rules and indexes but doesn't create a new Cloud Firestore instance with new configurations. I suppose a couple of examples is enough. 

Also, notice here that for YAML files parsing we may use some 3rd-party packages. In that case, we should review their pros and cons and state that we'll use yaml_map as for CI Integrations (tell why).

```suggestion
Before landscaping the feature, we should notice that the update feature is preliminary custom as implies changes to the Metrics CLI and is specific for Metrics project purposes.
```